roles/apt-repos/tasks/main.yml

```diff
@@ -3,13 +3,13 @@
   apt_key:
     url: "{{ item.key_url }}"
     validate_certs: "{{ item.validate_certs|default(omit) }}"
-  with_items: repos
+  with_items: "{{ repos|default([]) }}"
```

Comment:
We could define it there, but also yes, standardizing the syntax. Ansible 2.x will warn about A) raw variables in a with_items, and 2) it warns that the silent skipping of these tasks when the variable isn't defined is going away, and eventually they'll be errors. SO I was just going through and protecting every one, I didn't think of adding a default in the role itself.